pwnlib/shellcraft/templates/amd64/pushstr.asm

```diff
@@ -22,12 +21,11 @@ Args:
         extend = '\xff'
     else:
         extend = '\x00'
-%>
-
+%>\
```

Comment:
I vote we find some way to work around the extra newlines and never ever have to do this kind of stuff.